conan/tools/cmake/cmake.py

```diff
@@ -209,7 +209,12 @@ def install(self, build_type=None, component=None, cli_args=None, stdout=None, s
             arg_list.extend(["--component", component])
         arg_list.extend(self._compilation_verbosity_arg)
 
-        do_strip = self._conanfile.conf.get("tools.cmake:install_strip", check_type=bool)
+        deprecated_install_strip = self._conanfile.conf.get("tools.cmake:install_strip", check_type=bool)
+        if deprecated_install_strip:
+            self._conanfile.output.warning("The 'tools.cmake:install_strip' configuration is deprecated, "
+                                           "use 'tools.install:strip' instead.")
+
+        do_strip = self._conanfile.conf.get("tools.install:strip", check_type=bool)
```

Comment:
We should still listen to the deprecated strip conf, and fall back to the new one otherwise

Reply:
Done [013fd85](https://github.com/conan-io/conan/pull/18429/commits/013fd85671688745f0bcf2c3b1a1f26856155efe)